HaxeManual/05-expressions.tex

```diff
@@ -615,6 +615,7 @@ \section{throw}
 \section{cast}
 \label{expression-cast}
 
+The `cast` keyword changes the perceived type of an expression without changing its contents.
```

Comment:
Please use `\expr{cast}` instead of Markdown syntax. I'm also not sure the definition is strictly correct… Perhaps "reinterprets" instead of "changes"?